Dockerfile

```diff
@@ -1,13 +1,23 @@
-FROM alpine
+# BUILDER
 
-RUN apk update
+FROM golang:1.11 AS builder
+RUN go version
 
-RUN apk add nodejs npm git
+COPY  . /go/src/github.com/gravitational/slackbot
+WORKDIR /go/src/github.com/gravitational/slackbot
 
-ADD . /bot
+RUN set -x && \
+    go get github.com/golang/dep/cmd/dep && \
+    dep ensure -v
 
-RUN rm -rf /bot/.git
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o slackbot .
 
-WORKDIR /bot
+# RUNTIME
+FROM scratch
 
-ENTRYPOINT ["bin/hubot","-a","slack"]
+WORKDIR /bot/
+
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
```

Comment:
Use our debian-tall image, it contains ca-certificates.crt

Reply:
I went with the `scratch` image cause I didn't really need the rest of the OS (like shell and libraries) thus it would just be unneeded overhead.

I just really need the certs because of the Slack connection which would otherwise fail complaining about the Slack certificates.

Reply:
And our debian-tall is what you need actually, almost like a scratch and with 11MB size.